_zh-cn/overviews/scala3-book/introduction.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 导言
+type: chapter
+description: This page begins the overview documentation of the Scala 3 language.
+num: 1
+previous-page: 
+next-page: 
+
+scala3: true
+partof: scala3-book
+overview-name: "Scala 3 — Book"
+layout: multipage-overview
+permalink: "/zh-cn/scala3/book/:title.html"
```

Comment:
Is this property needed?

Reply:
These were deliberately added, which are related to the rendering of the page. You may notice:

https://github.com/scala/docs.scala-lang/blob/3b614bee449537d8e0b9b567ce7e36137eb7416f/_config.yml#L96-L106

Without messing up the global config file, I decided to add this snippet to each translation page, which should be more maintainable.